pkg/reconcilermanager/controllers/util.go

```diff
@@ -85,7 +85,11 @@ func hydrationEnvs(sourceType string, gitConfig *v1beta1.Git, ociConfig *v1beta1
 }
 
 // reconcilerEnvs returns environment variables for namespace reconciler.
-func reconcilerEnvs(clusterName, syncName string, syncGeneration int64, reconcilerName string, reconcilerScope declared.Scope, sourceType string, gitConfig *v1beta1.Git, ociConfig *v1beta1.Oci, helmConfig *v1beta1.HelmBase, pollPeriod, statusMode string, reconcileTimeout string, apiServerTimeout string, requiresRendering bool) []corev1.EnvVar {
+func reconcilerEnvs(clusterName, syncName string, syncGeneration int64,
```

Comment:
I think it's time to refactor these parameters, perhaps into an options struct. We can do that as a separate change if your prefer

Reply:
will do in a separate change.

Reply:
https://github.com/GoogleContainerTools/kpt-config-sync/pull/1037